bin/minifollowups/pycbc_injection_minifollowup

```diff
@@ -102,7 +110,10 @@ def sort_injections(args, inj_group, missed):
         sorter = dec_chirp_dist.argsort()
         return missed[sorter]
 
-    raise ValueError(f'Invalid injection sorting method "{args.distance_type}"')
+    raise ValueError(
```

Comment:
The real suggestion was to check this (`if not args.hasattr('distance type') or args.distance type not in legal_distance_types`) before trying to do anything else, on the principle 'error as soon as possible'.  I guess it doesn't make much difference overall, but putting the input check first would avoid eg weird errors from `if 'optimal_snr' in args.distance_type`